packages/bot-web-ui/src/pages/server-side-bot/add-bot/form.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import { useFormikContext } from 'formik';
+import { observer, useStore } from '@deriv/stores';
+import { useDBotStore } from 'Stores/useDBotStore';
+import QSInput from './inputs/qs-input';
+import QSInputLabel from './inputs/qs-input-label';
+import QSCheckbox from './inputs/qs-toggle-switch';
+import ContractTypeSelect from './selects/contract-type';
+import DurationTypeSelect from './selects/duration-type';
+import SymbolSelect from './selects/symbol';
+import TradeTypeSelect from './selects/trade-type';
+import { STRATEGIES } from './config';
+import { TConfigItem, TFormData, TShouldHave } from './types';
+import './add-bot.scss';
+
+const QuickStrategyForm = observer(() => {
+    const { ui } = useStore();
+    const { quick_strategy } = useDBotStore();
+    const { selected_strategy, setValue, form_data } = quick_strategy;
+    const config: TConfigItem[][] = STRATEGIES[selected_strategy]?.fields;
+    const { is_desktop } = ui;
+    const { values, setFieldTouched, setFieldValue } = useFormikContext<TFormData>();
+    const { current_duration_min_max } = quick_strategy;
+
+    const [isEnabledToggleSwitch, setIsEnabledToggleSwitch] = React.useState(false);
+
+    React.useEffect(() => {
+        window.addEventListener('keydown', handleEnter);
+        let data: TFormData | null = null;
+        try {
+            data = JSON.parse(localStorage.getItem('qs-fields') ?? '{}');
+        } catch {
+            data = null;
+        }
+        setIsEnabledToggleSwitch(!!data?.boolean_max_stake);
+
+        return () => {
+            window.removeEventListener('keydown', handleEnter);
```

Comment:
Can we replace the above logic of adding listeners to the `window` globally by adding `onKeyDown={handleEnter}` to the React node `<div className='ssb-add__body__content__form__group' key={group_index} ` in jsx itself?

Reply:
We will refactor the Form in the next phase, this form is copy of current quick strategy. Due to time limitation we won't be able to address it now @maryia-deriv, I'm keeping a note on this.